### PR TITLE
Update gas cost

### LINKED ISSUE #164: Update gas costs to Byzantium
Gas costs are still set at Homestead. Need to apply Spurious Dragon and Tangerine Whistle changes.

### PR DESCRIPTION
**Pull request by hugo-dc**
Updated gas costs (based on ethereumjs-vm costs).

Fixes #164.

### CODE REVIEW

File: opcodes.js [hunk 1]
```diff
@@ -135,21 +135,21 @@ const codes = {
...
-  0xa0: ['LOG', 0, 2, 0],
-  0xa1: ['LOG', 0, 3, 0],
-  0xa2: ['LOG', 0, 4, 0],
-  0xa3: ['LOG', 0, 5, 0],
-  0xa4: ['LOG', 0, 6, 0],
+  0xa0: ['LOG', 375, 2, 0],
+  0xa1: ['LOG', 375, 3, 0],
+  0xa2: ['LOG', 375, 4, 0],
+  0xa3: ['LOG', 375, 5, 0],
+  0xa4: ['LOG', 375, 6, 0],
```
Comment (axic) [inline opcodes.js]: I wonder if it is properly charging for the data too?

Comment (axic) [reply opcodes.js]: Can somebody check this?

**Comment by lrettig** [comment]
I tried running this against the tests in https://github.com/ethereum/tests/tree/develop/src/GeneralStateTestsFiller/stLogTests but they all depend on `call` which is broken pending #246

File: opcodes.js (hunk 1 above)
Comment (lrettig) [reply opcodes.js]: As discussed just now, `evm2wasm` does not charge for the data

**Comment by lrettig** [comment]
Both `evm2wasm` and `hera` are currently metering for the interface methods. We need to turn off one or the other. evm2wasm does not have complete gas metering logic for dynamic metering for opcodes like `LOG`, so in the short term we should probably continue to do metering in hera, which means disabling metering for interface opcodes in evm2wasm.

**Comment by axic** [comment]
I think we should close this as it may take a while doing this properly.

**Pull request closed without merging**